src/formatDate.js

```diff
@@ -8,7 +8,35 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.slice(3);
+  const newSeparator = toFormat.slice(3);
```

Comment:
The `slice` method is being used incorrectly here. `slice` is being called on an array with a single element, which will return an array, not the separator string. You should directly access the separator using the index `3` instead of using `slice`.